docs/admin/authorization/node.md

```diff
@@ -0,0 +1,75 @@
+---
+assignees:
+- timstclair
+- deads2k
+- liggitt
+- ericchiang
+title: Using Node Authorization
+---
+
+* TOC
+{:toc}
+
+Node authorization is a special-purpose authorization mode that specifically authorizes API requests made by kubelets.
+
+## Overview
+
+Kubelets must be able to read the API resources that describe the pods they are supposed to run,
+read secrets, configmaps, persistent volume claims and persistent volumes related to those pods,
+and write API objects that report the status of their pods, and of the node the kubelet is running on.
+
+The Node authorizer tracks which pods are associated with which nodes, and authorizes 
+kubelets to read just the resources related to their pods.
+
+In order to be authorized, kubelets must use a credential that identifies them as 
+being in the `system:nodes` group, with a username of `system:node:<nodeName>`.
+This group and user name format match the identity created for each kubelet as part of 
+[kubelet TLS bootstrapping](/docs/admin/kubelet-tls-bootstrapping/).
+
+To enable the Node authorizer, start the apiserver with `--authorization-mode=Node`.
+
+To limit the API objects kubelets are able to write, enable the [NodeRestriction](/docs/admin/admission-controllers#NodeRestriction) admission plugin by starting the apiserver with `--admission-control=...,NodeRestriction,...`
+
+## Migration considerations
+
+### Kubelets outside the `system:nodes` group
+
+Kubelets outside the `system:nodes` group would not be authorized by the `Node` authorization mode,
+and would need to continue to be authorized via whatever mechanism currently authorizes them.
+The node admission plugin would not restrict requests from these kubelets.
+
+### Kubelets with undifferentiated usernames
+
+In some deployments, kubelets have credentials that place them in the `system:nodes` group,
+but do not identify the particular node they are associated with (have a username that does not match the `system:node:...` format).
```

Comment:
suggested edit: "...they are associated with, such as having a username that does not match the `system:node:...` format."